timesketch/frontend-ng/src/components/Scenarios/SearchChip.vue

```diff
@@ -0,0 +1,56 @@
+<!--
+Copyright 2023 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <v-chip x-small @click="search(queryString)">
+    {{ displayName }}
+  </v-chip>
```

Comment:
UX nit: Is there a way to keep the chip highlighted when it was clicked and the search query is in the search box? This would allow the user to understand easily where the current search came from.

Reply:
Yes, that would be great! I thought it wasn't possible but I figured it out. Added highlight color to the selected chip:

<img width="880" alt="Screenshot 2023-07-07 at 15 33 25" src="https://github.com/google/timesketch/assets/316362/400481e8-9985-4497-a089-440c6a65236f">